templates/footer_country_selector.twig

```diff
@@ -29,7 +29,7 @@
             <ul aria-label="{{ __('Sites starting with the letter .' ~ initial ~ '.', 'planet4-master-theme') }}" role="list">
                 {% for country in countries %}
                 {% set main_lang, lang_count = country.lang|first, country.lang|length %}
-                <li>
+                <li role="listitem">
```

Comment:
Based on the [Mozilla's best practices page for lists](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listitem_role#best_practices)